hexgame/game.py

```diff
@@ -27,8 +27,30 @@ def __init__(
         self.current_player: Player = self.player_1
 
     def _play(self) -> None:
-        # player  move
-        self.current_player.play(self.board)
+        moved = False
+        trials_cap = 10
+        trials_num = 0
+        while not moved:
+            trials_num += 1
+            try:
+                # player  move
+                chosen_move: tuple[int, int, Color] = self.current_player.play(
+                    self.board
+                )
+                self.board._play(chosen_move)
+                moved = True
+            except ValueError as e:
```

Comment:
it woul have been nice to catch each individual specific error that can happen, as i don't remember what we expect the issues to be

Reply:
and also this way we can reraise any error that is not one of the known ones. Right now it is a bir riskt that we are try catching anything wrong thar can happen in there, don't you think?